README.md

```diff
@@ -1,14 +1,43 @@
-# CoreFoundation
+CoreFoundation
+==============
 
-FFI based wrappers for a subset of core foundation: various bits of CFString, CFData, CFArray, CFDictionary are available. Not that useful on its own but a useful building block for writing ffi wrappers of other macOS libraries.
+FFI based wrappers for a subset of core foundation: various bits of CFString, CFData, CFArray, CFDictionary are available.
 
 Although the CF collection classes can store arbitrary pointer sized values this wrapper only supports storing CFTypes.
 
 The CF namespace has the raw FFI generated method calls but it's usually easier to use the wrapper classes: `CF::String`, `CF::Date`, `CF::Array`, `CF::Dictionary`, `CF::Boolean` which try to present a rubyish view of the world (for example `CF::Array` implements `Enumerable`)
 
 These implement methods for creating new instances from ruby objects (eg `CF::String.from_string("hello world")`) but you can also pass build them from an `FFI::Pointer`).
 
-## Converting
+Preferences interface
```

Comment:
These need to be ## modern markdown

Reply:
Updated these along with the documentation update for memory management which was required since we now call `release` inside the object's finaliser. https://github.com/chef/corefoundation/pull/10/commits/bc4e7aaa0a701602eb5add582f5287b37a1789b7